example/lib/widgets/notice_banner.dart

```diff
@@ -20,6 +20,15 @@ class NoticeBanner extends StatelessWidget {
         foregroundColor = const Color(0xFF072100),
         backgroundColor = const Color(0xFFB8F397);
 
+  const NoticeBanner.informational({
+    super.key,
+    required this.text,
+    required this.url,
+    required this.sizeTransition,
+  })  : icon = Icons.info_outline,
+        foregroundColor = const Color(0xFF072100),
+        backgroundColor = Colors.lightBlue;
```

Comment:
.lightBlue is a bit hard on the eyes, can we use something different like lightBlueAccent please?

![Unbenannt](https://github.com/fleaflet/flutter_map/assets/34318751/e2d52da6-4f58-4dac-8528-07dcead335cf)